frontend/src/components/okrs/styles.scss

```diff
@@ -0,0 +1,48 @@
+.okr-container {
+    margin-left: 15px;
+    margin-bottom: 15px;
+    width: 285px;
+    height: 175px;
+    border: 2px solid #ff9900;
+    border-radius: 10px;
+    padding: 8px 15px;
+    display: grid;
+    grid-template-columns: 1fr, 1fr, 30px;
+    grid-template-rows: 1fr, 1fr, 100px;
+    grid-template-areas:
+            "n n re"
```

Comment:
If you come back to this code in a month, I don't think you'll remember that grid area name.